src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -169,6 +169,7 @@ DlgPrefInterface::DlgPrefInterface(QWidget * parent, MixxxMainWindow * mixxx,
     checkBoxScaleFactorAuto->hide();
     comboBoxScaleFactor->hide();
     labelScaleFactor->hide();
+    QApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
```

Comment:
`This attribute must be set before Q(Gui)Application is constructed.`
https://doc.qt.io/qt-5/qt.html
Looks like this should go in main.cpp before creating MixxxApplication.

Reply:
Good catch, thanks!